easybuild/tools/module_naming_scheme/categorized_hmns.py

```diff
@@ -83,6 +83,16 @@ def det_modpath_extensions(self, ec):
 
         return self.categorize_paths(basepaths)
 
+    def det_user_modpath_extensions(self, ec):
+        """
+        Determine user module path extensions, if any. As typical users are not expected to have many local modules,
+        further categorizing them using module classes is considered overkill. Thus, we are using a plain hierarchical
+        scheme for user modules instead.
+
+        Examples: Compiler/GCC/4.8.3 (for GCC/4.8.3 module), MPI/GCC/4.8.3/OpenMPI/1.6.5 (for OpenMPI/1.6.5 module)
+        """
+        return super(CategorizedHMNS, self).det_modpath_extensions(ec)
```

Comment:
this is exactly the same as the default implementation? what's the point of defining it then, just let inheritance take care of it?